components/arm/universalrobots/ur.go

```diff
@@ -487,6 +487,20 @@ func (ua *URArm) GoToInputs(ctx context.Context, goal []referenceframe.Input) er
 	return ua.MoveToJointPositions(ctx, positionDegs, nil)
 }
 
+// Geometries returns the list of geometries associated with the resource, in any order. The poses of the geometries reflect their
+// current location relative to the frame of the resource.
+func (ua *URArm) Geometries(ctx context.Context) ([]spatialmath.Geometry, error) {
+	inputs, err := ua.CurrentInputs(ctx)
+	if err != nil {
+		return nil, err
+	}
+	gif, err := ua.model.Geometries(inputs)
+	if err != nil {
+		return nil, err
+	}
+	return gif.Geometries(), nil
```

Comment:
wondering since each arm driver uses the same code if we should make this into a helper function in `arm.go`? I believe we should be able to accomplish this using the interface methods on arm rather than private struct attributes

Reply:
I agree with this.

When I do 3316 that will change the arm geometries impl pretty significantly; I'm more inclined to just make that change as part of that ticket. Would you be good with adding a TODO referencing that ticket?

Reply:
Yeah I'm good with that